Editor/CoreAPI/GameLiftComputeManager.cs

```diff
@@ -57,20 +71,26 @@ public GameLiftComputeManager(CoreApi coreApi, IAmazonGameLiftWrapper wrapper)
                 var registerComputeResponse =
```

Comment:
Why can't we return this direclty as part of the `Response.Ok`? Wondering what the need for the additional `RegisterComputeResponse` wrapper exactly is?

Reply:
This method has been changed alot in another PR, don't want to change anything until there is a merge.

Reply:
To deal with errors properly within our try catches, we need a proper method to capture the data and display it in the errors. With these wrappers it solves that issue and also follows the convention of the rest of the project.